queries.sh

```diff
@@ -0,0 +1,117 @@
+# Testing parent/child, remove before merging
+
+curl -X PUT localhost:9206/quiz?pretty=true -H 'Content-Type: application/json' -d  @- <<'EOF'
+{
+  "mappings": {
+    "question": {
+      "properties": {
+        "id_field": {
+          "type": "keyword"
+        },
+        "content": {
+          "type": "text"
+        },
+        "comment_type": {
+          "type": "join",
+          "relations": {
+            "question": "answer"
+          }
+        }
+      }
+    }
+  }
+}
+EOF
+
+curl -X PUT localhost:9206/quiz/_doc/1?pretty=true -H 'Content-Type: application/json' -d  @- <<'EOF'
+{
+  "id_field": "1",
+  "content": "2+3?",
+  "comment_type": "question"
+}
+EOF
+
+curl -X PUT localhost:9206/quiz/_doc/2?pretty=true -H 'Content-Type: application/json' -d  @- <<'EOF'
+{
+  "id_field": "2",
+  "content": "3+4?",
+  "comment_type": "question"
+}
+EOF
+
+
+curl -X PUT 'localhost:9206/quiz/_doc/3?routing=1&pretty=true' -H 'Content-Type: application/json' -d  @- <<'EOF'
+{
+  "id_field": "3",
+  "content": "3!",
+  "comment_type": {
+    "name": "answer",
+    "parent": "1"
+  }
+}
+EOF
+
+
+curl -X PUT 'localhost:9206/quiz/_doc/4?routing=1&pretty=true' -H 'Content-Type: application/json' -d  @- <<'EOF'
+{
+  "id_field": "4",
+  "content": "4!",
+  "comment_type": {
+    "name": "answer",
+    "parent": "1"
+  }
+
+}
+EOF
+
+# fails with
+#          "type" : "document_missing_exception",
+#          "reason" : "[question][3]: document missing",
+curl -X POST 'localhost:9206/_bulk/?pretty=true' -H 'Content-Type: application/json' -d  '
+{ "update": { "_id": "3", "_index": "quiz", "_type": "question" }  }
```

Comment:
Why do we use `_type` here?

Reply:
I think I used an old manual while doing those experiments. They're soon to be removed either way.